core/templates/core/component/record_tracker_label.html

```diff
@@ -1,17 +1,14 @@
-<div class="record_tracker_label">
-
-    {% if creation_timestamp or change_timestamp  %}
+{% if creation_timestamp or change_timestamp %}
+    <div class="record_tracker_label">
         (
-    {% endif %}
 
-    {% if creation_timestamp %}
-        {{ subject }} created {{ creation_timestamp }} by {{ creation_user }}.
-    {% endif %}
-    {% if change_timestamp and change_timestamp != creation_timestamp  %}
-        Last changed {{ change_timestamp }} by {{ change_user }}
-    {% endif %}
+        {% if creation_timestamp %}
+            {{ subject }} created {{ creation_timestamp }} by {{ creation_user }}.
+        {% endif %}
+        {% if change_timestamp and change_timestamp != creation_timestamp %}
```

Comment:
Because the `change_timestamp` is set to `model_utils.default_current_timestamp` (same as `creation_timestamp`) , there will be a minor difference between the two values when a record is created (see example of a comment created below).

![image](https://user-images.githubusercontent.com/5488106/218428153-d39a4071-361c-4559-83ac-72525fd1e368.png)


Perhaps a solution is to remove `default=model_utils.default_current_timestamp` from all recref models?

Reply:
`default=model_utils.default_current_timestamp` is backup plan for `Recref` model,  I will update `update_current_user_timestamp` instead

Reply:
created new issues, fix that later